portfolio/src/main/webapp/index.html

```diff
@@ -7,8 +7,16 @@
     <script src="script.js"></script>
   </head>
   <body bgcolor = "#66819c">
+    <div id="navbar">
+	<ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="lucygallery.html">Lucy Gallery</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#suprise">Suprise</a></li>
+      </ul>
+    </div>
```

Comment:
FYI: There is a `<nav>` element specifically meant for such things.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
```suggestion
    <nav id="navbar">
	<ul>
        <li><a href="index.html">Home</a></li>
        <li><a href="lucygallery.html">Lucy Gallery</a></li>
        <li><a href="#projects">Projects</a></li>
        <li><a href="#suprise">Suprise</a></li>
      </ul>
    </nav>
```